app/models/concerns/patient_session_state_machine_concern.rb

```diff
@@ -101,7 +101,7 @@ def consent_given?
     def consent_refused?
       return false if no_consent?
 
-      consents.any?(&:response_refused?)
+      consents.all?(&:response_refused?)
```

Comment:
Really? When I was putting together the consent details page I thought that this is correct ... is there going to be a `consent_conflict?` ?

Reply:
Yeah. `consent_refused?` is when all consents were refused. `consent_conflicts?` is when some were refused and some were given.